app.py

```diff
@@ -1,52 +1,145 @@
 from flask import Flask, jsonify, request
 from models import insert_genero, insert_filme, insert_usuario, insert_diretor
-from models import  update_genero, update_filme, update_usuario, update_diretor
+from models import update_genero, update_filme, update_usuario, update_diretor
 from models import delete_genero, delete_filme, delete_usuario, delete_diretor
-from models import select_genero, select_filme, select_usuario, select_diretor
+from models import select_genero, select_filme, select_usuario, select_diretor, select_generoid
 from validacao import valida_genero, valida_filme, valida_usuario, valida_diretor
-from serializadores import genero_from_db, genero_from_web, filme_from_db, filme_from_web, usuario_from_db, usuario_from_web, diretor_from_db, diretor_from_web
+from serializadores import *
 
 app = Flask (__name__)
 
 
 # Criando genero, filme, usuário e diretor
-@app.route ("/generos", methods=["POST"])
-def addGenero():
-    genero = genero_from_web (**request.json)
-    if valida_genero (**genero):
-        insert_genero (**genero)
-        return jsonify (genero_from_db (genero))
+@app.route("/generos", methods=["POST"])
+def addgenero():
+    genero = genero_from_web(**request.json)
+    if valida_genero(**genero):
+        insert_genero(**genero)
+        return jsonify(genero_from_db(genero))
     else:
-        return jsonify ({"Gênero inválido!!"})
+        return jsonify({"Gênero inválido!!"})
 
 
-@app.route ("/filmes", methods=["POST"])
-def addFilme():
-    filme = filme_from_web (**request.json)
-    if valida_filme (**filme):
-        insert_filme (**filme)
-        return jsonify (filme_from_db (filme))
+@app.route("/filmes", methods=["POST"])
+def addfilme():
+    filme = filme_from_web(**request.json)
+    if valida_filme(**filme):
+        insert_filme(**filme)
+        return jsonify(filme_from_db (filme))
```

Comment:
Pra todos os inserts a ideia é retornar os itens do banco de dados que foram criados.
```suggestion
    filme = filme_from_web(**request.json)
    if valida_filme(**filme):
        id_filme = insert_filme(**filme)
        filme_inserido = get_filme(id_filme)
        return jsonify(filme_from_db (filme_inserido))
```